base_comment_template/migrations/15.0.1.1.0/pre-migration.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    for item in env["base.comment.template"].sudo().search([]):
```

Comment:
You can't use the model as well, as it's not loaded in the registry. It should be pure SQL.